rhaptos/cnxmlutils/xsl/test/title.cnxml.html

```diff
@@ -10,10 +10,18 @@
   <div
     data-type='document-title'
   >Document testing the use of the title tag</div>
-<?chapter-toc label="Learning Objectives"?>
-<?cnx.eoc class="section-summary" title="Section Summary"?>
-<?cnx.eoc class="conceptual-questions" title="Conceptual Questions"?>
-<?cnx.eoc class="problems-exercises" title="Problems & Exercises"?>
+  <cnx-pi
+    data-type='chapter-toc'
+  >label="Learning Objectives"</cnx-pi>
+  <cnx-pi
+    data-type='cnx.eoc'
+  >class="section-summary" title="Section Summary"</cnx-pi>
```

Comment:
@pumazi Shouldn't the custom tag be <cnx-pi data-type="cnx.eoc" class="section-summary" title="Section Summary"></cnx-pi> ?